lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1242,6 +1237,39 @@ def get_service_name_from_init_system(self, service):
             else:
                 return service
 
+    def get_magma_services(self) -> List[str]:
+        """
+        Returns a list of all services managed by magmad and additionally
+        magmad itself, openvswitch-switch and sctpd.
+        """
+        non_magmad_services = [
+            'magmad',
+            'sctpd',
+        ]
+
+        systemd_only_magma_services = [
+            'openvswitch-switch',
+        ]
+
+        docker_only_magma_services = [
+            'connectiond',
+            'monitord',
+            'redirectd',
+            'td-agent-bit',
+        ]
+
+        raw_magmad_yml = self.exec_command_output('cat /etc/magma/magmad.yml')
```

Comment:
Why not use:
```
with open('/etc/magma/magmad.yml', 'r') as f:
   yml.load(f, ...)
```

Reply:
The `magmad.yml` is fetched from the dev VM and does not exist the one on the test VM. The `with open(...)` statement would cause an error here.